resources/authgear/templates/en/web/authflowv2/settings_identity_email.html

```diff
@@ -0,0 +1,89 @@
+{{ template "authflowv2/__settings_page_frame.html" . }}
+
+{{ define "page-navbar" }}
+    {{ template "authflowv2/__navbar.html"
+    (dict
+    "BackHref" (call $.MakeURL "/settings")
+    "Title" (translate "v2.page.settings-identity-email.default.navbar-title" nil)
+    )
+    }}
+{{ end }}
+
+{{ define "page-content" }}
+
+<div class="mt-8 settings-content flex flex-col gap-5">
+    {{ $hasMultiple := gt (len .CheckedEmails) 1 }}
+
+    <span>
+        {{ range $index, $element := .CheckedEmails }}
+            <div class="settings-identity-email-block">
+                <div class="settings-identity-email-indent with-icon">
+                    <div class="settings-identity-email-text">{{ $element.Email }}</div>
+                    {{ if $hasMultiple }}
+                    <a 
+                        class="settings-item__forward_arrow material-icons"
+                        href="/settings/identity/email">
+                        arrow_forward_ios
+                    </a>
+                    {{ else }}
+                    <span class="body-text--lg"><a 
+                        href="/settings/identity/email/edit">
+                        {{ translate "v2.page.settings-identity.default.edit-label" . }}
+                    </a></span>
+                    {{ end }}
+                </div>
+                <div class="settings-identity-email-indent pt-1">
+                    {{ if $element.Verified }}
+                    <div class="settings-identity-email-subtext success">
+                    {{ translate "v2.page.settings-identity-email.default.verified-label" . }}
+                    {{ else }}
+                    <div class="settings-identity-email-subtext failure">
+                    {{ translate "v2.page.settings-identity-email.default.not-verified-label" . }}
+                    {{ end }}
+                    </div>
```

Comment:
All the if-branches should be a complete expression.
i.e. the close tag of the div should be place in the branch

```html
{{ if some_condition }}
  <div></div>
{{ else }}
  <div></div>
{{ end }}
```

Don't be lazy